app/commands/fever_api/read_feeds_groups.rb

```diff
@@ -13,9 +13,11 @@ def call(authorization:, **params)
     private
 
     def feeds_groups(authorization)
-      scoped_feeds = authorization.scope(FeedRepository.in_group)
+      scoped_feeds = authorization.scope(FeedRepository.list)
```

Comment:
I haven't done any extensive research. I don't own iOS devices so even trying to find an Android App that supports Fever still it took me ages to find Read You. I've been using Press so far but it's really dead and I have to copy the APK from device to device each time I switch for years now. 

Do you know of any maintained compatible Android clients?

I cannot imagine this breaks anyone really though, as groups seem to be a standard feature of all clients supporting the Fever API. However I acknowledge it might change a usage pattern for some clients perhaps, since in the case of browsing "ungrouped" feeds in your client, if your client was supporting them at all now you have an "Ungrouped" group instead now. However since Stringer doesn't really allow group management in the first place I would be very surprised this really breaks anyones workflow.

Reply:
Hmm, maybe that's a case for implementing a more compatible API. I'm not too sure what is common these days, though. At any rate, we can let it be for now and see if anything comes up.

Reply:
Yeah, implementing a more popular API could be useful. Though even the Fever API already kind off has more features than a minimalist app like Stringer provides. I can imagine more modern alternatives are not any lower on additional features which might get hard to map out.

Looking at Read You and what I vaguely remember from looking at apps I think FreshRSS, TinyTiny-RSS and Nextcloud News would be candidates to look at as for an API to copy.